opensrp-app/src/main/java/org/smartregister/sync/helper/PlanIntentServiceHelper.java

```diff
@@ -38,7 +40,10 @@
 
     private final PlanDefinitionRepository planDefinitionRepository;
     private final AllSharedPreferences allSharedPreferences;
-    protected static final Gson gson = new GsonBuilder().registerTypeAdapter(LocalDate.class, new DateTypeConverter()).create();
+    protected static Gson gson = new GsonBuilder().registerTypeAdapter(DateTime.class, new DateTimeTypeConverter("yyyy-MM-dd"))
```

Comment:
Not sure how this works with `2019-03-31T0000`. Could you confirm? Is the `T0000` part essential and/or ignored during conversion?

Found the format in CampaignTest

Reply:
This for plans serialization, and deserialization, plans use Dates and thus dont need the time part. Since the execution period was changed from Localdate to DateTime, this is necessary to avoid side effects

Reply:
Cool